cloudbuild.yaml

```diff
@@ -7,7 +7,7 @@ steps:
   entrypoint: bash
   env:
   - DRIVER_IMAGE_REGISTRY=us-central1-docker.pkg.dev/k8s-staging-images/dra-example-driver
-  - DRIVER_IMAGE_TAG=$COMMIT_SHA
+  - DRIVER_IMAGE_TAG=$_GIT_TAG # _GIT_TAG is injected by Prow
```

Comment:
Nit: `_GIT_TAG` doesn't come from Prow, it's from the builder tool.

Magic.

Let's try it, we can always iterate further.

/lgtm
/approve